exercises/04.2-Apply-several-classes/README.md

```diff
@@ -24,4 +24,4 @@ Replace the class property `spades` of the `<div>` with the class `heart` and lo
 
 ## Expected result:
 
-![Poker Card Heart](https://github.com/4GeeksAcademy/css-tutorial-exercises-course/blob/master/.learn/assets/04.2-1.png?raw=true)
+![Poker Card Heart](../../assets/04.2-1.png?raw=true)
```

Comment:
The path is wrong, the correct path should be: `../../.learn/assets/04.2-1.png?raw=true`